easybuild/easyblocks/l/llvm.py

```diff
@@ -925,6 +926,12 @@ def test_step(self):
             if LooseVersion(self.version) >= LooseVersion('19'):
                 self._set_gcc_prefix()
                 self._create_compiler_config_file(self.cfg_compilers, self.gcc_prefix, self.final_dir)
+            # For nvptx64 tests, find out if 'ptxas' exists in $PATH. If not, ignore all nvptx64 test failures
+            pxtas_path = shutil.which('ptxas')
```

Comment:
Here we can use the `which` integrate in easybuild which is already used also in the RPATH section